src/shapes/IText/ITextKeyBehavior.ts

```diff
@@ -78,7 +77,7 @@ export abstract class ITextKeyBehavior<
     this.hiddenTextarea.addEventListener('keyup', this.onKeyUp.bind(this));
     this.hiddenTextarea.addEventListener('input', this.onInput.bind(this));
     this.hiddenTextarea.addEventListener('copy', this.copy.bind(this));
-    this.hiddenTextarea.addEventListener('cut', this.copy.bind(this));
```

Comment:
cut is fine as copy, since it has to delegate the removal of text to onInput to keep the logic centralized.

Reply:
I will centralize, but firing copy when cutting is not good